theano/compile/tests/test_profiling.py

```diff
@@ -11,94 +13,96 @@
 from theano.ifelse import ifelse
 
 
-def test_profiling():
+class Test_profiling(unittest.TestCase):
+
+    def test_profiling(self):
```

Comment:
Now there is a few test for profiling. Can you add a docstring with the goal of the tests. I think it would be useful.